imposm/dump-pg.sh

```diff
@@ -30,15 +30,15 @@ if [ -z ${PBF_LOCATION+x} ]
 then
      echo "Mandatory parameter '-i' is missing";
      echo "Usage dump-pg.sh -i osm_pbf_location [-p postgresqlPort] [-c postgresqlContainerName]"
-     echo "  -i Location of the PBF file to import"
+     echo "  -i Location of the PBF file to import, or directory containing multiple files with .pbf extensions, to support multi-file import"
      echo "  -p Local port to use for the PostgreSQL container, defaults to $PG_PORT_DEFAULT"
      echo "  -v Version of the Kartoza PostGIS Image to be used, defaults to $PG_IMAGE_VERSION_DEFAULT"
      echo "  -c Name of the container used for the PostgreSQL container, defaults to $PG_CONTAINER_DEFAULT"
      echo "  -r Remove the database container and its data after import, either true or false, defaults to $PG_CONTAINER_REMOVE"
 
      exit 1
 fi
-if [ ! -f "$PBF_LOCATION" ]; then
+if [ [! -f "$PBF_LOCATION"] && ! [-d "$PBF_LOCATION"] ]; then
```

Comment:
Extra spaces may be necessary in the test cases next to the square brackets
[! -f "$PBF_LOCATION"] && ! [-d "$PBF_LOCATION"] - > [ ! -f "$PBF_LOCATION" ] && ! [ -d "$PBF_LOCATION" ]

Reply:
Done, thanks!